bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gpi/GpiCB.java

```diff
@@ -88,17 +88,6 @@ public void apply(Magnitude magnitude) {
         config.putParameter(systemName,
                             StringParameter.getInstance(InstConstants.INSTRUMENT_NAME_PROP,
                                     Gpi.INSTRUMENT_NAME_PROP));
-        final ISPObsComponent targetcomp = SPTreeUtil.findTargetEnvNode(getObsComponent().getContextObservation());
-        if (targetcomp != null) {
-            final TargetObsComp toc = (TargetObsComp) targetcomp.getDataObject();
-            if (toc != null) {
-                if (toc.getTargetEnvironment().getBase() != null) {
-                    ImList<Magnitude> magnitudes = toc.getTargetEnvironment().getBase().getMagnitudesJava();
-                    magnitudes.filter(new MagnitudeFilter(MagnitudeBand.H$.MODULE$)).headOption().foreach(new MagnitudeSetter(Gpi.MAG_H_PROP));
-                    magnitudes.filter(new MagnitudeFilter(MagnitudeBand.I$.MODULE$)).headOption().foreach(new MagnitudeSetter(Gpi.MAG_I_PROP));
-                }
-            }
-        }
```

Comment:
Yeah I talked to Carlos and he thought it was probably left over and could be deleted. It doesn't do anything and it's been there since we imported OCS into GitHub.